vassal-app/src/main/java/VASSAL/build/module/ModuleExtension.java

```diff
@@ -416,6 +413,7 @@ public String getName() {
     return name;
   }
 
+  @Override
```

Comment:
I think you can delete this function, since you're providing an identical copy in the parent class now.

Reply:
All fixed